verilog/formatting/tree_unwrapper.cc

```diff
@@ -499,8 +499,12 @@ void TreeUnwrapper::InterChildNodeHook(const SyntaxTreeNode& node) {
     case NodeEnum::kBlockItemStatementList:
       LookAheadBeyondCurrentNode();
       break;
-    default:
+    default: {
+      if (Context().DirectParentIs(NodeEnum::kMacroArgList)) {
+        StartNewUnwrappedLine(PartitionPolicyEnum::kFitOnLineElseExpand);
+      }
```

Comment:
This change seems to address //comments that appear under the `kMacroArgList` level.  Can you add a few test cases that cover this?  Or (if it doesn't break any existing tests), can revert this until later.

Reply:
Done. Added tests to `formatter_test.cc` and `tree_unwrapper_test.cc`